docs/Release-Process.md

```diff
@@ -45,6 +45,14 @@ branch to read only whilst the release action runs.
 > [!NOTE]
 > Admin permission is needed to complete this task.
 
+### Wait for the `VERSION` bump PR payload publish to complete
```

Comment:
So if the version bump PR fails CI, rerunning it is the way to go?

Reply:
Yeah, it's either re-run it, or if a fix is required then doing it before the need to mess about with draft releases is easier.